functional-test/src/main/java/org/zanata/page/WebDriverFactory.java

```diff
@@ -136,6 +144,29 @@ private WebDriver configureChromeDriver() {
                                 capabilities)));
     }
 
+    /**
+     * Returns a File pointing to a chromedriver binary, searching the PATH
+     * if necessary. If the property 'webdriver.chrome.driver' points to an
+     * executable, uses that.  Otherwise, searches PATH for the specified
+     * executable.  Searches for 'chromedriver' if the property is null or
+     * empty.
+     * @return a File pointing to the executable
+     * @deprecated use ChromeDriverService.createDefaultService() if you can
+     */
+    @Deprecated
+    private File getChromeDriver() {
+        String exeName = PropertiesHolder.getProperty("webdriver.chrome.driver");
+        if (exeName == null || exeName.isEmpty()) {
+            exeName = CommandLine.find("chromedriver");
+        } else if (!new File(exeName).canExecute()) {
+            exeName = CommandLine.find(exeName);
```

Comment:
If the deprecation warning is that bothersome, I suppose I could steal ExecutableFinder.java's source code (it's ASL 2.0).

I don't know why Selenium insists on a File pointing to the exact executable...